tests/pub-data-urls_top-level-content/OPS/package.opf

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<package xmlns="http://www.idpf.org/2007/opf" xmlns:epub="http://www.idpf.org/2007/ops" version="3.0" xml:lang="en" unique-identifier="pub-id">
+  <metadata xmlns:dc="http://purl.org/dc/elements/1.1/">
+    <dc:coverage>Publication Resources</dc:coverage>
+    <dc:creator>Matthew Chan</dc:creator>
+    <dc:date>2021-12-03</dc:date>
+    <dc:description>
+      The reading system must display an img (where src is a data URL) as part of the Top-level Content
+      Document in which it is embedded, rather than opening it as thought it were its own Top-level Content Document.
+    </dc:description>
```

Comment:
Optional: is there a more explicit way in which you can describe what the expected/aberrant behavior here would be? It feels to me like it would be hard to *see* whether the image is top-level rather than merely happening to reflow to the next page. If you can find a way to clarify this, please put it in both the description and the paragraph on page 1.

Reply:
Happy holidays, dlazin! (I've been rushing for non-epub related deadlines, so super apologies for delay.)

New commit below cleans up sentence case cleans up some nits.
It also clarifies that the svg is absolutely positioned at the bottom of page 1 in a fixed layout epub. If the svg _does_ appear as top-level content, then the RS has definitely done something unexpected.